assets/app/views/project.html

```diff
@@ -4,12 +4,12 @@
       <tasks></tasks>
       <div ng-if="!renderOptions.showGetStarted" class="actions pull-right">
         <div class="btn-group">
-          <label class="btn btn-default" ng-model="$parent.overviewMode" btn-radio="'tiles'" title="Tile View">
+          <a href="" class="btn btn-default" ng-model="$parent.overviewMode" btn-radio="'tiles'" title="Tile View">
```

Comment:
I think it should be like this, based on http://getbootstrap.com/components/#btn-groups
Also, switching to the pf topology icon
```html
<div class="btn-group" role="group" aria-label="View Options">
  <button type="button" class="btn btn-default">
    <span class="fa fa-th-list" aria-hidden="true"></span>
  </button>
  <button type="button" class="btn btn-default">
   <span class="pficon pficon-topology" aria-hidden="true"></span>
  </button>
</div>
```